.vscode/settings.json

```
@@ -0,0 +1,17 @@
+{
+    "editor.codeActionsOnSave": {
+        "source.fixAll.markdownlint": true
```

I'm not sure I want this as I use autosave. And some lints are better handled silencing rather than fixing them.

---

I meant on a per-case basis, but the sticking point is the autosave - I don't think insta-fixing every lint that shows up while editing will be a good experience

---

we can try without it

```suggestion
        "source.fixAll.markdownlint": false
```